### PR TITLE
Removed redundant "const" and "new" assignments & Formatted code.

### PR DESCRIPTION
**Pull request by rkinabhi**

### CODE REVIEW

File: jsonexample/lib/built_value/built_complex_object.g.dart [hunk 1]
```diff
@@ -19,7 +19,7 @@ part of 'built_complex_object.dart';
 // ignore_for_file: unnecessary_new
 
 Serializer<BuiltComplexObject> _$builtComplexObjectSerializer =
-    new _$BuiltComplexObjectSerializer();
+    _$BuiltComplexObjectSerializer();
```
Comment (RedBrogdon) [inline jsonexample/lib/built_value/built_complex_object.g.dart]: Did you edit these files by hand (the ones ending with `.g.dart`), or rerun the built_value source generation routine?

File: AUTHORS [hunk 2]
```diff
@@ -4,3 +4,4 @@
 #   Name/Organization <email address>
 
 Google Inc.
+Abhijeeth Padarthi <rkinabhi@gmail.com>
```
Comment (RedBrogdon) [inline AUTHORS]: nit: Need a newline here.

File: jsonexample/lib/built_value/built_complex_object.g.dart (hunk 1 above)
Comment (rkinabhi) [reply jsonexample/lib/built_value/built_complex_object.g.dart]: I've done these edits manually. read and understood how json_serializable works.  will revert changes made on the generated files. :) 👍

**Comment by rkinabhi** [comment]
@RedBrogdon I have reverted the changes made on the generated files.

**Comment by RedBrogdon** [comment]
Thanks. This LGTM, and I'd like to merge it.

Have you signed the Google contributor agreement (https://cla.developers.google.com)?

**Comment by rkinabhi** [comment]
Yes, I've signed it. :)

**Comment by RedBrogdon** [comment]
Hmm. You're right, I can see it right there in the pre-merge checks. We don't have an automatic label set up the way some other Flutter repos do, though. I'll need to ask someone about it.

**Comment by Hixie** [comment]
I spoke to Abhijeeth Padarthi by e-mail and they let me know that they are ok with relicensing this to BSD for consistency with the rest of the repo. (E-mail dated Thu, Nov 21, 2019 at 5:48 PM.)

**Pull request merged**